central/scannerdefinitions/handler/handler.go

```diff
@@ -447,9 +472,6 @@ func (h *httpHandler) openMostRecentV4File(t updaterType, updaterKey, fileName s
 		if err != nil {
 			return nil, err
 		}
-		if targetFile == nil {
```

Comment:
why is this now deleted?

Reply:
Because `openFromArchive()` returns error when it cannot return the proper file so the error check should be sufficient enough.